.github/workflows/CICD.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         - { os: macos-10.15   , ocaml-version: 4.03.0 }
         - { os: macos-10.15   , ocaml-version: 4.02.3 }
         - { os: macos-10.15   , ocaml-version: 4.01.0 }
-        - { os: ubuntu-18.04  , ocaml-version: 4.12.0 }
+        - { os: ubuntu-18.04  , ocaml-version: 4.12.0+musl+static+flambda }
```

Comment:
Don't remove the regular 4.12 build. Should the static build be added to CI, it should not replace the regular 4.12 build.

Reply:
Thanks; I didn't spot that.

I expect that after the discussion on hackers we'll make some other changes and not merge this, although this PR is useful for starting that larger discussion.